packages/riipen-ui/src/components/List.jsx

```diff
@@ -10,23 +10,42 @@ class List extends React.Component {
     children: PropTypes.node,
 
     /**
-     * List of additional classes to apply to this component.
+     * Array or string of additional CSS classes to use.
+     *
+     * @type {string | Array}
      */
-    classes: PropTypes.array
+    classes: PropTypes.oneOfType([PropTypes.array, PropTypes.string])
   };
 
   static defaultProps = {
     classes: []
   };
 
+  handleClick = (child, idx) => {
+    return () => {
+      if (child.disabled) return;
+      this.setState({
+        activeItemIndex: idx
+      });
+    };
+  };
+
   render() {
     const { classes, children } = this.props;
 
-    const className = clsx(classes);
+    const className = clsx(classes, "list");
 
     return (
       <React.Fragment>
-        <ul className={className}>{children}</ul>
+        <ul className={className}>
+          {children.map((child, idx) => {
+            return React.cloneElement(child, {
+              onClick: this.handleClick(child, idx),
+              key: `${idx}`,
+              ...child.props
+            });
+          })}
```

Comment:
Should be able to remove al of this in favor of `{children}` once the `disable` issue is dealt with.